ledger-core/virtual/integration/method_test.go

```diff
@@ -174,16 +175,49 @@ func TestVirtual_Method_WithExecutor(t *testing.T) {
 func TestVirtual_Method_WithExecutor_ObjectIsNotExist(t *testing.T) {
 	t.Log("C4974")
 
+	mc := minimock.NewController(t)
+
 	server, ctx := utils.NewServer(nil, t)
 	defer server.Stop()
 	server.IncrementPulse(ctx)
 
 	var (
 		objectLocal  = server.RandomLocalWithPulse()
 		objectGlobal = reference.NewSelf(objectLocal)
+		outgoing     = server.RandomLocalWithPulse()
 	)
 
-	Method_PrepareObject(ctx, server, payload.Ready, objectGlobal)
+	Method_PrepareObject(ctx, server, payload.Ready, reference.NewSelf(server.RandomLocalWithPulse()))
+
+	executeDone := server.Journal.WaitStopOf(&execute.SMExecute{}, 1)
+
+	expectedError := throw.E("no state on object after readyToWork", struct {
+		ObjectReference string
+		State           object.State
+	}{
+		ObjectReference: objectGlobal.String(),
+		State:           object.Missing,
+	})
+
+	typedChecker := server.PublisherMock.SetTypedChecker(ctx, mc, server)
+	typedChecker.VStateRequest.Set(func(request *payload.VStateRequest) bool {
+		assert.Equal(t, request.Object, objectGlobal)
+		return true
```

Comment:
you can manually create expected VStateReport, and send it with server.SendPayload

Reply:
I am testing stepWaitObjectReady behavior and how it returns correct error

Reply:
well you don't create object, you test execute behavior. So VStateRequest/VStateReport is outside of your scope and tested elsewhere